go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/chain-selectors v1.0.27
 	github.com/smartcontractkit/chainlink-automation v0.8.0
-	github.com/smartcontractkit/chainlink-ccip v0.0.0-20241021132654-e5f3ecb77638
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20241022182841-dfbaf574a68f
```

Comment:
Should this commit be on the main branch?

Reply:
yes, ty.